src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_consumer_system_v2/test_compressor_system_v2_results/consumer_system_v2.json

```diff
@@ -3789,6 +3789,63 @@
                     "2026-01-01 00:00:00"
                 ]
             }
+        },
+        "cbc0427cf665c504a8081b89cf167ee3": {
+            "co2": {
+                "name": "co2",
+                "quota": {
+                    "regularity": [
+                        1.0,
+                        1.0
+                    ],
+                    "timesteps": [
+                        "2022-01-01 00:00:00",
+                        "2026-01-01 00:00:00"
+                    ],
+                    "typ": "STREAM_DAY",
+                    "unit": "NOK/d",
+                    "values": [
+                        0.0,
```

Comment:
0 co2?

Reply:
0 quota for co2, 85.8 tons co2 (rate below)